keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/GaussianVertex.java

```diff
@@ -135,4 +136,20 @@ public PartialDerivatives forwardModeAutoDifferentiation(Map<Vertex, PartialDeri
             return PartialDerivatives.withRespectToSelf(this.getId(), this.getShape());
         }
     }
+
+    /**
+     * @return a set of parent edges with edges connecting to vertices representing mu and sigma hyperparameters annotated accordingly.
+     */
+    @Override
+    public Set<GraphEdge> getParentEdgesInDotFormat() {
+        Set<GraphEdge> edges = super.getParentEdgesInDotFormat();
+
+        GraphEdge muEdge = new GraphEdge(this, mu);
+        edges.stream().filter(muEdge::equals).findFirst().get().appendToDotLabel("mu");
+
+        GraphEdge sigmaEdge = new GraphEdge(this, sigma);
+        edges.stream().filter(sigmaEdge::equals).findFirst().get().appendToDotLabel("sigma");
+
+        return edges;
+    }
```

Comment:
You might be able to use the same annotations I added for the Protobuf stuff if you want parent names (will save you having to annotate all the vertices for a second time?).  We should also then be able to pull this code out in to the DotWriter stuff explicitly.  May require a bit of refactoring to pull out some of the code to read annotations, but I'm happy to help with that.

Reply:
Done it now :)